Packs/Sigma/Playbooks/playbook-Sigma_-_Query_using_Sigma_rules.yml

```diff
@@ -0,0 +1,1344 @@
+id: Sigma - Query using Sigma rules
```

Comment:
In your instance the playbook is in quiet mode, did you disable it manually?

Reply:
Don't think so, maybe for testing? ill put it back on.

Reply:
I see it's in quiet mode here as well, is that wrong?